merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/http/MerlinBindings.java

```diff
@@ -181,14 +187,15 @@ private void getResourceTypes(final Context ctx) {
     }
   }
 
-  private void getSimulationResults(final Context ctx) {
+  private void getSimulationResults(final Context ctx) throws Unauthorized {
```

Comment:
What is the reasoning for throwing this `Unauthorized` exception at this top level instead of catching it below the try block in each handler function?